src/index.js

```diff
@@ -63,8 +51,13 @@ Jutsu.propTypes = {
   displayName: PropTypes.string,
   onMeetingEnd: PropTypes.func,
   loadingComponent: PropTypes.object,
+  errorComponent: PropTypes.object,
   containerStyles: PropTypes.object,
-  jitsiContainerStyles: PropTypes.object
+  jitsiContainerStyles: PropTypes.object,
+  configOverwrite: PropTypes.object,
+  interfaceConfigOverwrite: PropTypes.object,
```

Comment:
Those were previously passed using the `options` props which no longer exists.
Any prop not used by the component it self will be passed to the hook